src/key_config.rs

```diff
@@ -27,6 +27,8 @@ impl From<KeyConfigType> for u8 {
     }
 }
 
+// Should probably be removed as it should manage bus endianness while
```

Comment:
how is this managing bus endianness? Need to be able to construct a Key/SlotConfig from bytes right?